epochStart/bootstrap/process.go

```diff
@@ -39,7 +39,8 @@ import (
 
 var log = logger.GetOrCreate("epochStart/bootstrap")
 
-const timeToWait = time.Minute
+// DefaultTimeToWait represents the default timespan until requested data needs to be received from the connected peers
+const DefaultTimeToWait = time.Minute
```

Comment:
maybe rename to `DefaultTimeToWaitForRequestedData`

Reply:
done